CODE_OF_CONDUCT.md

```diff
@@ -106,23 +106,27 @@ Violating these terms may lead to a permanent ban.
 ### 4. Permanent Ban
 
 **Community Impact**: Demonstrating a pattern of violation of community
-standards, including sustained inappropriate behavior,  harassment of an
+standards, including sustained inappropriate behavior, harassment of an
 individual, or aggression toward or disparagement of classes of individuals.
 
-**Consequence**: A permanent ban from any sort of public interaction within
-the community.
+**Consequence**: A permanent ban from any sort of public interaction within the
+community.
 
 ## Attribution
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage],
-version 2.0, available at
-https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+version 2.1, available at
+[https://www.contributor-covenant.org/version/2/1/code_of_conduct.html][v2.1].
```

Comment:
Looks like the URL is not formatted correctly and contains the wrong text, should be something like

```suggestion
[here](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html).
```

or 
```
[version 2.1](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html)
```

Reply:
That's a valid way of using Markdown links.
It is very useful if the link is used multiple times, so you can use `[v2.1]` (in this case) rather than repeating the same link multiple times. 🙂


You can check [this readme](https://raw.githubusercontent.com/Crystal741/pink_details/main/README.md) as a reference as well.